js/src/modals/DeployContract/DetailsStep/detailsStep.js

```diff
@@ -37,6 +37,7 @@ export default class DetailsStep extends Component {
     onParamsChange: PropTypes.func.isRequired,
     onInputsChange: PropTypes.func.isRequired,
 
+    balances: PropTypes.object,
```

Comment:
Same comment here as above (last time, I promise) - rather than trying arbitrary spacings between props, just sort and condense. Easy to find without understanding what the developer had in mind as a category system.